src/components/scalable-when-grabbed.js

```diff
@@ -1,21 +1,14 @@
 import { paths } from "../systems/userinput/paths";
 
-const grabberToPath = {
-  "player-right-controller": paths.actions.rightHand.scaleGrabbedGrabbable,
-  "player-left-controller": paths.actions.leftHand.scaleGrabbedGrabbable,
-  cursor: paths.actions.cursor.scaleGrabbedGrabbable
-};
-
 AFRAME.registerComponent("scalable-when-grabbed", {
   tick: function() {
-    const grabber = this.el.components.grabbable.grabbers[0];
-    if (!grabber) return;
-
     const userinput = AFRAME.scenes[0].systems.userinput;
-    const deltaScale = userinput.get(grabberToPath[grabber.id]);
+    const interaction = AFRAME.scenes[0].systems.interaction;
+    if (interaction.state.rightRemote.held !== this.el) return;
+    const deltaScale = userinput.get(paths.actions.cursor.scaleGrabbedGrabbable);
     if (!deltaScale) return;
 
-    this.el.object3D.scale.addScalar(deltaScale).clampScalar(0.01, 1000);
+    this.el.object3D.scale.addScalar(deltaScale).clampScalar(0.1, 100);
```

Comment:
These were the original clamp values but I changed them two weeks ago without anyone noticing and for no particular reason.